godot-core/src/meta/class_id.rs

```diff
@@ -256,21 +230,21 @@ impl ClassIdEntry {
 /// `Cow`-like enum for class names, but with C strings as the borrowed variant.
 enum ClassIdSource {
     Owned(String),
-    Borrowed(&'static CStr),
+    Borrowed(&'static str),
 }
```

Comment:
This can now become `Cow<'static, str>`, moving `to_string_name` into a global fn.